src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
```

Comment:
for cases like this you can use the `background` shorthand property.
you can read more about it here https://developer.mozilla.org/en-US/docs/Web/CSS/background